leaderboard.py

```diff
@@ -125,94 +120,94 @@ def update_filtered_leaderboard(tab, summary_lang: str, source_lang: str,
+      filter_chosen = value != ANY_LANGUAGE
+      return gr.Dataframe(visible=not filter_chosen), gr.Dataframe(
+          visible=filter_chosen)
+
     with gr.Tab(LeaderboardTab.SUMMARIZATION.value):
-      with gr.Accordion("Filter", open=False) as summarization_filter:
-        with gr.Row():
-          languages = [
-              language.name.capitalize() for language in lingua.Language.all()
-          ]
-          summary_language = gr.Dropdown(
-              choices=languages,
-              value=DEFAULT_FILTER_OPTIONS["summary_language"],
-              label="Summary language",
-              interactive=True)
-
-        with gr.Row():
-          filtered_summarization = gr.DataFrame(
-              headers=["Rank", "Model", "Elo rating"],
-              datatype=["number", "str", "number"],
-              value=lambda: load_elo_ratings(
-                  LeaderboardTab.SUMMARIZATION, DEFAULT_FILTER_OPTIONS[
-                      "summary_language"]),
-              elem_classes="leaderboard")
-
-      summary_language.change(fn=update_filtered_leaderboard,
-                              inputs=[
-                                  gr.State(LeaderboardTab.SUMMARIZATION),
-                                  summary_language,
-                                  gr.State(),
-                                  gr.State()
-                              ],
-                              outputs=filtered_summarization)
-
-      gr.Dataframe(headers=["Rank", "Model", "Elo rating"],
-                   datatype=["number", "str", "number"],
-                   value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION),
-                   every=LEADERBOARD_UPDATE_INTERVAL,
-                   elem_classes="leaderboard")
+      summary_language = gr.Dropdown(choices=SUPPORTED_LANGUAGES +
+                                     [ANY_LANGUAGE],
+                                     value=ANY_LANGUAGE,
+                                     label="Summary language",
+                                     interactive=True)
+
+      filtered_summarization = gr.DataFrame(
+          headers=["Rank", "Model", "Elo rating"],
+          datatype=["number", "str", "number"],
+          value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION,
+                                         ANY_LANGUAGE),
+          elem_classes="leaderboard",
+          visible=False)
+
+      original_summarization = gr.Dataframe(
+          headers=["Rank", "Model", "Elo rating"],
+          datatype=["number", "str", "number"],
+          value=lambda: load_elo_ratings(LeaderboardTab.SUMMARIZATION),
+          every=LEADERBOARD_UPDATE_INTERVAL,
+          elem_classes="leaderboard")
       gr.Markdown(LEADERBOARD_INFO)
 
+      summary_language.change(
+          fn=update_filtered_leaderboard,
+          inputs=[
+              gr.State(LeaderboardTab.SUMMARIZATION), summary_language,
+              gr.State(),
```

Comment:
it seems like it's src, target language, but it's summarization, so it should be empty and `gr.State()` does that?

i'm not sure but could you make that part clear? or maybe split the functions for each... etc

Reply:
yes. I needed to pass Gradio Component as inputs so I passed the empty State Component. I'll update it to gr.State(None)